messaging/kafka-avro-reactive-messaging/src/main/resources/docker-compose-strimzi.yaml

```diff
@@ -0,0 +1,44 @@
+version: '2'
```

Comment:
Is this file used?

Reply:
if you launch this app manually then this file is very useful.

Reply:
Then, please add in the README.md information about how to use it and run the application.